chain/neutrino.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/btcsuite/btcd/rpcclient"
 	"github.com/btcsuite/btcd/txscript"
 	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcwallet/chain/internal/rescan"
```

Comment:
Importing this package as `rescan` will cause some variables that are called the same to become shadowing. One more reason to not add those interfaces to a new sub package.